application/src/main/java/org/opentripplanner/street/model/vertex/StationEntranceVertex.java

```diff
@@ -0,0 +1,19 @@
+package org.opentripplanner.street.model.vertex;
+
+public class StationEntranceVertex extends OsmVertex {
+
+  private final String entranceName;
+
+  public StationEntranceVertex(double x, double y, long nodeId, String entranceName) {
+    super(x, y, nodeId);
+    this.entranceName = entranceName;
+  }
+
+  public String getEntranceName() {
+    return entranceName;
+  }
```

Comment:
I think this should be called `entranceCode` for the purpose we use. In the future, if we include entrances from GTFS with the same model, the name and code will be different things.

Reply:
Or perhaps just code.

Reply:
Why not ´publicCode' ?

Reply:
In GTFS terminology, it's called stop_code in stops.txt. I'm not a native english speaker, but public code doesn't make it more understandable for me vs code.

Reply:
`public` say it is used as information to the public. There are also other codes used for other things, like `privateCode` used by agencies for internal references.